packages/base/Tag/src/TagCheckable/TagCheckable.tsx

```diff
@@ -38,7 +44,14 @@ const TagCheckable = forwardRef<HTMLDivElement, Props>(function TagCheckable(
 
   return (
     <Tag
-      className={cx(className, classes.checkable)}
+      className={twMerge(
+        `hover:border-graphite-700 transition-all duration-[350ms] ease-in-out cursor-pointer`,
+        className,
+        checked && 'hover:border-red-500 hover:text-red-500',
+        hovered && 'border-graphite-700',
+        checked && hovered && 'border-red-500 text-red-500'
```

Comment:
I think we should move `className` to last, so it is possible to override all default styles from outside
```suggestion
        checked && 'hover:border-red-500 hover:text-red-500',
        hovered && 'border-graphite-700',
        checked && hovered && 'border-red-500 text-red-500'
        className,
```

Reply:
Not sure about this. In a case when someone passes for example `border-gree-600` as `className`, the border color will always be `green-600`, and there would not be a difference when the `hovered` or `checked` prop will be passed. 

On the other hand, a user can handle changing these styles himself. Are we ok with this?

Reply:
We usually try to make the classnames more strict, in that sense it would be better to not let classNames override the borders.

But I noticed that before this change, className was actually: `className={cx(className, classes.checkable)}`
I believe that changing that order might have breaking changes consequences, can you check if the previous styles for `classes.checkable` are always at last position? In specific, `classes.checkable` has some pretty complex selectors, I'm not sure if we are supporting all of them here:
```js
'&$checkable:not($disabled)': {
        cursor: 'pointer',
        '&:hover, &$hovered': {
          borderColor: palette.grey.dark,
          backgroundColor: palette.common.white,
          transition: `all ${transitions.duration.short}ms ${transitions.easing.easeInOut}`,
        },
        '&:focus': {
          backgroundColor: palette.common.white,
        },
        '&$green': {
          '&:hover, &$hovered': {
            borderColor: palette.red.main,
            color: palette.red.main,
          },
        },
      },
      ```